docs/book/features/block_volume.md

```diff
@@ -375,3 +375,69 @@ Cluster
             VolumeAttributes:      type=vSphere CNS Block Volume
         Events:                <none>
     ```
+
+### Statically Provision a Block Volume in Tanzu Kubernetes Grid Service<a id="static_volume_provisioning_tkg"></a>
+
+You can statically create a block volume in Tanzu Kubernetes Grid Service (TKGS) using an unused PersistentVolumeClaim (PVC) from the Supervisor cluster only if the latter satisfies the following conditions:
+
+- It is present in the same Supervisor cluster namespace where the TKGS was created
+
+- Not attached to a Pod in the Supervisor cluster or in any other TKGS
+
+Using static provisioning, a PVC created by one TKGS can also be re-used in another TKGS when the volume is no longer needed in the former one. In order to do so, you need to modify the `Reclaim policy` of the PV in the old TKGS to `Retain` and then delete the corresponding PVC. Follow the steps below to statically create a PVC in the new TKGS using the information from the leftover underlying volume.
+
+Steps to create a static PVC in a TKGS are as follows:
+
+1. Note the name of the PVC in the supervisor cluster. If you are re-using a PVC from an old TKGS, then this value can also be retrieved from the `volumeHandle` of the old PV object in the TKGS.
+
+2. Create a PersistentVolume with the `storageClassName` preferably set to the storage class name of your supervisor cluster PVC and the `volumeHandle` pointing to the value retrieved in step 1. If you are re-using a volume from another TKGS where it is no more required, delete the PVC and PV object from the old TKGS before creating a PV in the new TKGS.
+
+    ```yaml
+      apiVersion: v1
+      kind: PersistentVolume
+      metadata:
+        name: static-tkg-block-pv
+        annotations:
+          pv.kubernetes.io/provisioned-by: csi.vsphere.vmware.com
+      spec:
+        storageClassName: gc-storage-profile
```

Comment:
We also need to set the claim name to make sure that it binds to the PVC that we are about to create.

Reply:
It's not mandatory right? I just put up a minimal YAML for PV template which will bind to the PVC we create next. We can also set labels to make sure they bind

Reply:
I changed it anyways